x-pack/plugin/ml/src/main/java/org/elasticsearch/xpack/ml/aggs/frequentitemsets/FrequentItemSetsAggregator.java

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+package org.elasticsearch.xpack.ml.aggs.frequentitemsets;
+
+import org.elasticsearch.search.aggregations.Aggregator;
+import org.elasticsearch.search.aggregations.support.AggregationContext;
+import org.elasticsearch.search.aggregations.support.ValuesSourceConfig;
+import org.elasticsearch.xpack.ml.aggs.mapreduce.MapReduceAggregator;
+
+import java.io.IOException;
+import java.util.List;
+import java.util.Map;
+
+public final class FrequentItemSetsAggregator extends MapReduceAggregator {
+
+    protected FrequentItemSetsAggregator(
```

Comment:
```suggestion
    FrequentItemSetsAggregator(
```
`protected` doesn't make sense when a class is `final`